paddle/fluid/framework/operator.h

```diff
@@ -66,6 +67,7 @@ inline std::string GradVarName(const std::string& var_name) {
 }
 
 proto::VarType::Type GetDataTypeOfVar(const Variable* var);
+bool VarIsTensor(const Variable& var);
```

Comment:
why expose this? Seems not a good name for what it does

Reply:
This is used in ngraph_operator.cc, so it is shared here.